datasets/scripts/LCMS_Visualization.js

```diff
@@ -0,0 +1,225 @@
+
+var startYear = 1985;
+var endYear = 2020;
+var lossYearPalette = ['ffffe5', 'fff7bc', 'fee391', 'fec44f', 'fe9929',
+                       'ec7014', 'cc4c02'];
+var gainYearPalette = ['c5ee93', '00a398'];
+var durationPalette = ['BD1600', 'E2F400','0C2780'];
+
+var names_values_colors = {
+  'Land_Cover': {
+    'names': [
+       'Trees',
+       'Tall Shrubs & Trees Mix',
+       'Shrubs & Trees Mix',
+       'Grass/Forb/Herb & Trees Mix',
+       'Barren & Trees Mix',
+       'Tall Shrubs',
+       'Shrubs',
+       'Grass/Forb/Herb & Shrubs Mix',
+       'Barren & Shrubs Mix',
+       'Grass/Forb/Herb',
+       'Barren & Grass/Forb/Herb Mix',
+       'Barren or Impervious',
+       'Snow or Ice',
+       'Water',
+       'Non-Processing Area Mask'
+    ],
+    'colors': [
+       '005e00',
+       '008000',
+       '00cc00',
+       'b3ff1a',
+       '99ff99',
+       'b30088',
+       'e68a00',
+       'ffad33',
+       'ffe0b3',
+       'ffff00',
+       'AA7700',
+       'd3bf9b',
+       'ffffff',
+       '4780f3',
+       '1B1716'
+    ]
+  },
+  'Land_Use': {
+    'names': [
+       'Agriculture',
+       'Developed',
+       'Forest',
+       'Non-Forest Wetland',
+       'Other',
+       'Rangeland or Pasture',
+       'Non-Processing Area Mask'
+    ],
+    'colors': [
+       'efff6b',
+       'ff2ff8',
+       '1b9d0c',
+       '97ffff',
+       'a1a1a1',
+       'c2b34a',
+       '1B1716'
+    ]
+  },
+  'Change': {
+    'names': [
+       'Stable',
+       'Slow Loss',
+       'Fast Loss',
+       'Gain',
+       'Non-Processing Area Mask'
+    ],
+    'colors': [
+       '3d4551',
+       'f39268',
+       'd54309',
+       '00a398',
+       '1B1716'
+    ]
+  }
+};
+var lossYearViz = {min: startYear, max: endYear, palette: lossYearPalette};
+var gainYearViz = {min: startYear, max: endYear, palette: gainYearPalette};
+var durationViz = {min: 1, max: 5, palette: durationPalette};
+
+var lcViz = {min: 1, max: 15, palette: names_values_colors.Land_Cover.colors};
+var luViz = {min: 1, max: 7, palette: names_values_colors.Land_Use.colors};
+
+
+
+// #############################################################################
+// ### Define functions ###
+// #############################################################################
+
+/**
+ * Convert given code to year that number was present in the image.
+ */
+function getMostRecentChange(c, code){
+  return c.map(function(img) {
+    var yr = ee.Date(img.get('system:time_start')).get('year');
+    return ee.Image(yr).int16().rename(['year']).updateMask(img.eq(code))
+      .copyProperties(img,['system:time_start']);
+  });
+}
+
+
+// #############################################################################
+// ### Bring in LCMS annual outputs ###
+// #############################################################################
+
+var lcms = ee.ImageCollection(
+  'projects/lcms-292214/assets/Final_Outputs/2020-5/LCMS');
```

Comment:
please use the catalog collection name USFS/GTAC/LCMS/v2020-5

Reply:
I updated the collection path. @simonff  please review.